scripts/extractEdition.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from model import Edition
+from main import loadEnvFile
+from managers import DBManager
+
+
+#Extracting the edition number from the edition_statement column of the DRB database to fill out the edition column
+def main():
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    for edit in dbManager.session.query(Edition) \
+        .filter(Edition.edition_statement != None and Edition.languages[0]['language'] == 'English'):
+            editStatement = edit.edition_statement
+            editDict = {'first': 1, 'second': 2, 'third': 3, 'fourth': 4, 'fifth': 5, \
```

Comment:
This should probably use the `extract` method you used for the model! Just so we only have to maintain it in one place.